crates/rpc-replay/tests/test_replay_block.rs

```diff
@@ -26,13 +27,28 @@ async fn test_replay_block() {
     let provider = JsonRpcClient::new(HttpTransport::new(
         Url::parse(provider_url.as_str()).expect("Could not parse provider url"),
     ));
-    let state_reader = AsyncRpcStateReader::new(provider, BlockId::Number(block_with_txs.block_number - 1));
+    let block_id = BlockId::Number(block_with_txs.block_number - 1);
```

Comment:
`previous_block_id`?

Reply:
Not really sure. That was the value that was set for this test